data_pipeline/adjacency_mat.py

```diff
@@ -0,0 +1,89 @@
+from typing import List
+import penman
+
+INSTANCE_ARC_LABEL = ":instance"
+ROOT_LABEL = ':root'
+
+
+def get_concepts_to_triple_ids(g: penman.Graph):
+  """
+  Extract a dictionary concept -> triple id (where the 
+  concept is instanced).
```

Comment:
Why the new line 😆 or why here, in the middle of the paranthesis?

Reply:
typo